packages/babel-plugin-proposal-decorators/test/fixtures/2023-11-classes--to-es2015/replacement-static-decorator-initializer-this/exec.js

```diff
@@ -37,3 +41,4 @@ expect(methodThis).toBe(original);
 expect(accessorThis).toBe(replaced);
 expect(propertyThis).toBe(replaced);
 
+expect(classThis).toBe(replaced);
```

Comment:
Is this correct? This is `original` in `main`.

Reply:
This is correct, the class initializers are called on the replaced class, see step 41 of RS: ClassDefinitionEvaluation.